docs/contributing/setup.md

```diff
@@ -49,7 +58,7 @@ repository.
 After cloning the repository, the typical workflow to get up running
 is as follows:
 
-* Run `npm install` to get all required dependencies on your machine.
+* Run `yarn` to get all required dependencies on your machine.
 * Run `npm run build:dev` to create a development build of the app.
```

Comment:
Is there some `yarn` equivalent to `npm run`? Feels a little weird to use a mix of both, but maybe that's ok.

Reply:
`yarn foo` = `npm run foo`

Reply:
Oh snap, thanks for the hint @j-f1! That's gonna save some keystrokes! Will tidy up.